src/aap_eda/settings/default.py

```diff
@@ -617,6 +617,13 @@ def get_rulebook_process_log_level() -> RulebookProcessLogLevel:
 DIRECT_SHARED_RESOURCE_MANAGEMENT_ENABLED = settings.get(
     "DIRECT_SHARED_RESOURCE_MANAGEMENT_ENABLED", False
 )
+if DIRECT_SHARED_RESOURCE_MANAGEMENT_ENABLED:
+    RQ_PERIODIC_JOBS.append(
```

Comment:
@bzwei It needs to define the job_id to be a static value, otherwise we will have issues with multiple running schedulers.